.github/workflows/check.yml

```diff
@@ -23,35 +23,8 @@ on:
 permissions:
```

Comment:
Think about concurrency:

* Either tell people just don't
* For SLSA, put a random number at the end of the artifact, but that will complicate how we fetch the workflow artifact.

Reply:
Added a warning